packages/shared/src/components/filters/MyFeedHeading.tsx

```diff
@@ -55,16 +54,8 @@ function MyFeedHeading({
     <>
       {forceRefresh && (
         <Button
-          size={
-            shouldUseMobileFeedLayout && !isNewMobileLayout
-              ? ButtonSize.Small
-              : ButtonSize.Medium
-          }
-          variant={
-            isMobile && isNewMobileLayout
-              ? ButtonVariant.Tertiary
-              : ButtonVariant.Float
-          }
+          size={ButtonSize.Medium}
+          variant={isMobile ? ButtonVariant.Tertiary : ButtonVariant.Float}
```

Comment:
I understand this is not part of the supposed changes, but should we maybe store these values in a const, so whenever we need some changes in the future, we won't miss a button that would be misaligned?

Reply:
Hmm not sure I understand here, this is only used in this line so what other button could be missaligned?

Reply:
There is another button a bit below that has the same `size` and `variant` values/conditions.

Reply:
Hm, it has the same size and variant but the condition there is isLaptop, while here isMobile

Reply:
Ah, okay. Cool, feel free to ignore this one. Thank you!